Gemfile

```diff
@@ -8,7 +8,7 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 3.7.3"
+gem "jekyll", "~> 3.8.5"
```

Comment:
Is this version supported by GitHub Pages?

Reply:
We no longer rely on GH's pages build. We handle it ourselves.

Reply:
Looks like it supports up until 3.7.4. See https://pages.github.com/versions/

Reply:
Bummer

Reply:
So? Supported or not?